llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/metrics/LlapTaskSchedulerMetrics.java

```diff
@@ -276,6 +302,43 @@ private void getTaskSchedulerStats(MetricsRecordBuilder rb) {
         .addCounter(SchedulerPendingPreemptionTaskCount, pendingPreemptionTasksCount.value())
         .addCounter(SchedulerPreemptedTaskCount, preemptedTasksCount.value())
         .addCounter(SchedulerCompletedDagCount, completedDagcount.value());
+    daemonTaskLatency.forEach((k, v) -> rb.addGauge(v, v.getMean()));
```

Comment:
Not sure that I understand the purpose of this line (storing the DaemonLatencyMetric as an own gauge upon reset)...

Reply:
We need to add them as a gauge so it will show up on the counter /JMX list